Patch-Source: https://github.com/llvm/llvm-project/commit/d97981c98a70ebeaa8901f34921b0a69a068ff5d
--
From d97981c98a70ebeaa8901f34921b0a69a068ff5d Mon Sep 17 00:00:00 2001
From: Lang Hames <lhames@gmail.com>
Date: Thu, 16 Nov 2023 16:45:29 -0800
Subject: [PATCH] [ORC-RT] Add missing cstdint include.

This should have been included in b2bbe8cc1c7. Adding it should fix the bot
failures in https://lab.llvm.org/buildbot/#/builders/85/builds/20288
---
 compiler-rt/lib/orc/stl_extras.h | 1 +
 1 file changed, 1 insertion(+)

--- a/compiler-rt-16.0.6.src/lib/orc/stl_extras.h
+++ b/compiler-rt-16.0.6.src/lib/orc/stl_extras.h
@@ -13,6 +13,7 @@
 #ifndef ORC_RT_STL_EXTRAS_H
 #define ORC_RT_STL_EXTRAS_H
 
+#include <cstdint>
 #include <utility>
 #include <tuple>
 
